rosdep/python.yaml

```diff
@@ -7691,6 +7691,15 @@ python3-qtpy:
   gentoo: [dev-python/QtPy]
   nixos: [python3Packages.qtpy]
   ubuntu: [python3-qtpy]
+python3-qwt:
+  debian: [python3-qwt]
+  fedora:
+    pip:
+      packages: [PythonQwt]
+  opensuse:
+    pip:
+      packages: [PythonQwt]
```

Comment:
I think the preference is to put pip keys in a separate entry (`python3-qwt-pip`) to avoid collisions if the package becomes available on these platforms in the future. [Reference](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules).

Reply:
@wjwwood please advise what you think the right thing to do here is.

Reply:
Agreed.

Reply:
Thanks! Split off into `-pip` in 9a78bfe.
Do the ones with system packages need to be repeated with their pip packages in the `-pip` entry?

Reply:
I think that wouldn't hurt and is good for completeness.

Reply:
Added in c8e5f72